config/env/defaults.js

```diff
@@ -297,7 +297,7 @@ function getQueueDb() {
         password: password,
         port: port,
         database: database,
-        poolSize: process.env.QUEUE_DB_MAX_POOL_SIZE || 1,
+        max: process.env.QUEUE_DB_MAX_POOL_SIZE || 1,
```

Comment:
Why `max`? Wouldn't it be better `maxPoolSize`? I mean, it's clear when you see `process.env.QUEUE_DB_MAX_POOL_SIZE`, but without this "context", might be unclear.